lib/galaxy/model/mapping.py

```diff
@@ -103,7 +103,7 @@ def _build_model_mapping(engine, map_install_models, thread_local_log) -> Galaxy
 
 
 def init_models_from_config(
-    config: GalaxyAppConfiguration,
+    config: Type["GalaxyAppConfiguration"],
```

Comment:
```suggestion
    config: "GalaxyAppConfiguration",
```

You're sending in an object of that type - not the type itself I think - I think this is why the linting is failing now also.